### ISSUE TITLE
npm start not working in windows: The filename, directory name, or volume label syntax is incorrect.

### ISSUE THREAD

**Issue by anujjoshi63** [electron-react-boilerplate/electron-react-boilerplate#3279]
## Prerequisites

- [x] Using npm
- [x] Using an up-to-date [`main` branch](https://github.com/electron-react-boilerplate/electron-react-boilerplate/tree/main)
- [x] Using latest version of devtools. [Check the docs for how to update](https://electron-react-boilerplate.js.org/docs/dev-tools/)
- [x] Tried solutions mentioned in [#400](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/400)
- [x] For issue in production release, add devtools output of `DEBUG_PROD=true npm run build && npm start`

## Expected Behavior
App should have started in dev mode.

## Current Behavior
" The filename, directory name, or volume label syntax is incorrect. " was getting thrown. 
All three of cmd, powershell, cmder were giving same error.

## Steps to Reproduce

1. npm start

## Possible Solution (Not obligatory)
I fixed this issue by **replacing** all the scripts of format `abc:xyz` to `abc-xyz` in scripts of package.json.
Same change is required for `start:preload` and `start:main` in .erb\configs\webpack.config.renderer.dev.ts
Apparently `abc:xyz` made windows think it was a drive or something.

## Context
I was trying to get the boilderplate setup. I had no issues with `npm install`. I followed the Readme and went on to run `npm start` and came across this issue. I did not make any changes to any file in the cloned repo.

## Your Environment

- Node version : v16.14.0
- electron-react-boilerplate version or branch : main branch v4.6.0. (cloned on the same day when this issue was created.)
- Operating System and version : windows 10 version 21H2 (OS build 19044.1826)
- Link to your project : -

**Comment by anujjoshi63** [comment]
TLDR: issue fixed by replacing all the scripts of format abc:xyz to abc-xyz in scripts of package.json.
Apparently abc:xyz made windows think it was a drive or something.

e.g. replace all occurrences `start:main` by `start-main` and so on till `npm start` works.

**Comment by amilajack** [comment]
Interesting, I don't think this is an issue for other windows users. Do you have a unique windows configuration?

**Comment by anujjoshi63** [comment]
No @amilajack, I haven't done any unique configs. 
I did some research after your comment, I found out [here](https://exerror.com/the-filename-directory-name-or-volume-label-syntax-is-incorrect/) that the issue was with npm. 
So i did this:
`npm install -g npm@latest`
and now it works as expected with colons as well. 
Thanks though

**Comment by abdullahIsa** [comment]
> npm install -g npm@latest

i agree, did this and worked
